src/main/java/com/powsybl/openloadflow/network/LfAction.java

```diff
@@ -334,6 +334,7 @@ private void applyGeneratorChange(LfNetworkParameters networkParameters) {
         if (!generator.isDisabled()) {
             double newTargetP = generatorChange.isRelative() ? generator.getTargetP() + generatorChange.activePowerValue() : generatorChange.activePowerValue();
             generator.setTargetP(newTargetP);
```

Comment:
I was about to merge the PR but I have a doubt, why you don't change these lines too in a `LfContingency`:
````
        Set<LfBus> generatorBuses = new HashSet<>();
        for (LfGenerator generator : lostGenerators) {
            **generator.setTargetP(0);** -> initial targetP to zero too ?
            LfBus bus = generator.getBus();
            generatorBuses.add(bus);
            generator.setParticipating(false);
            generator.setDisabled(true);
````

Reply:
indeed, but adding setInitialTergetP(0) would improve code clarity and won't harm, I am adding it.